src/schema-validator.ts

```
@@ -0,0 +1,11 @@
+import Ajv, {JSONSchemaType} from "ajv";
+import { parse } from "yaml";
+import { TemplateConfig } from "./types";
+
+export default (schema: JSONSchemaType<TemplateConfig>, input: string) => {
+    const ajv = new Ajv() 
+    const validate = ajv.compile(schema)
+    const valid = validate(parse(input))
+    if (!valid) console.log(validate.errors)
```

We should probably return the validation errors to give to teams so they know what's wrong with their configuration 😄 WDYT? 

We could return a 
```
{
  valid: boolean,
  errors?: string[]
}

type that's easy to present in a PR comment or something similar 😄